### ISSUE TITLE
Keyboard accessibility - web interface

### ISSUE THREAD

**Issue by llune** [jitsi/jitsi-meet#5992]
## Description
---
https://beta.meet.jit.si/

![image](https://user-images.githubusercontent.com/6073336/79231490-6d13ce80-7e66-11ea-8b3a-4cd13edef073.png)

On the web version.

## Current behavior
---
The div element to create a room is not keyboard accessible.

`<div class="welcome-page-button" id="enter_room_button">Créer</div>`

## Expected Behavior
---

Creating a room should be keyboard accessible.

## Possible Solution
---

Use a button element or role="button" and tabindex="0" on the div + add keyboard support (a button is supposed to work with Space or Enter).

## Priority
---

As it's possible to create a room while hitting Enter in the form (which I did not know) maybe it's less urgent than the other issues concerning accessibility.

**Comment by Echolon** [comment]
Hi,
The team is aware of accessibility. Any reports are now grouped in the pinned issue #6090.

---
*I'm a community contributor and not directly affiliated with the Jitsi team.*

**Comment by luixxiul** [comment]
I'd agree with the opener- button element should actually be styled with `<button>`. This needs a fix.

**Comment by luixxiul** [comment]
This too:

https://github.com/jitsi/jitsi-meet/blob/638fdf03708604fdc2e97ab612613550adbfd922/react/features/toolbox/components/web/ToolbarButton.js#L76

**Comment by luixxiul** [comment]
I noticed that this issue should cover not only elements on `WelcomePage.web.js` but also ones inside `#new-toolbox` on the video conference page.

**Comment by luixxiul** [comment]
Reopened to track this issue and its fix specifically.

**Comment by shane215** [comment]
When I get time, I’d like to confirm if this issue was fixed by big PR #8921.
